contracts/contracts_test.go

```diff
@@ -0,0 +1,86 @@
+package contracts
+
+import (
+	"encoding/json"
+	"testing"
+	"testing/fstest"
+
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/manifest"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/nef"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetFS(t *testing.T) {
+	c, err := GetFS()
+	require.NoError(t, err)
+	require.Equal(t, len(fsContracts), len(c))
```

Comment:
suggest to also test `They're returned in the order they're supposed to be deployed starting
from NNS` statement from docs. We can also check exact (i guess) result contents since embedded files are now presented in the repo

ok to be done later